source/UIAHandler/customAnnotations.py

```diff
@@ -32,17 +19,31 @@
 GUID for the annotation type.
 """
 
+import dataclasses
+from typing import (
+	ClassVar,
+	Dict,
+)
+
+from comtypes import (
+	GUID,
+	byref,
+)
+
+import winVersion
+
 
-@dataclass
+@dataclasses.dataclass

```

Comment:
I think the previous import strategy should remain

```suggestion
@dataclass
```